### PR TITLE
base functionality

### PR DESCRIPTION
**Pull request by jigar288**
Merging version 1 branch with master

### CODE REVIEW

**Comment by clee231** [comment]
I'd suggest not including all your `node_modules` if possible...
Your dependencies should already be documented in your `package.json`.

**Comment by anarmawala** [comment]
Are we still using AWS Amplify for the functionality? If not, we should be able to remove them safely.

**Comment by bmiddha** [comment]
> Are we still using AWS Amplify for the functionality? If not, we should be able to remove them safely.

Just check in the react app. I don't think we'll be using Amplify.

File: src/calendarImports/Config.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+export const CtaBusConfig = {
```
Comment (bmiddha) [inline src/calendarImports/Config.ts]: @jigar288 the ctabus, ctatrain, transit, darksky configs among other thigns aren't needed.
please clean up this file

File: src/util/Config.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
+export const CtaBusConfig = {
```
Comment (bmiddha) [inline src/util/Config.ts]: @jigar288 There are 2 of the same files in this code base. figure out which one is not needed and remove it.
`src/util/Config.ts` `src/calendarImports/Config.ts`

**Pull request merged**